Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -408,12 +406,6 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
     {
         this._showTimestamps = WI.settings.showConsoleMessageTimestamps.value;
         this._currentSessionOrGroup.element.classList.toggle("timestamps-visible", this._showTimestamps);
-        if (this._showTimestamps) {
```

Comment:
if you're inclined, i think we can also get rid of `_showTimestamps` and instead just inline `WI.settings.showConsoleMessageTimestamps.value` in those uses